docs/getting-started/installing/ha-master.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Highly available Kontena Master
+---
+
+# Running Kontena master in HA setup
+
+Kontena master can be run in HA setup where multiple instances of the Kontena master form one highly available logical master.
+
+## Requirements for HA
+
+- Master nodes:
+  - Any modern Linux distribution with support for Docker and docker-compose. CoreOS and Ubuntu are more heavily tested.
+  -
+- MongoDB cluster:
+  - 3 node MongoDB replica-set is recommended
+
+    Using provided services such as Compose.io can give huge operational benefits with some additional costs.
+
+    **Note:** When using DBaaS services, make sure you can run the database in the same datacenter as you run your masters. If not there will be severe latencies in the database operations that could cause un-anticipated failures.
+
+## Running Kontena masters
+
+Running multiple master instances is as simple as running single instance. The key is to point all the master instances to the same database cluster and to use exact same configuration for all master instances.
+
+### Setting Kontena master HA with docker-compose
+
+If you are setting up Kontena master using docker-compose you can use following configuration:
+
+```yaml
+version: '2'
+services:
+  master:
+    image: kontena/server:latest
```

Comment:
Should we mention something about using "latest"? (might be a better idea to use 1.x here)